spring-pulsar-docs/src/main/asciidoc/pulsar.adoc

```diff
@@ -69,19 +69,6 @@ template.newMessage(msg)
 ----
 ====
 
-====== Custom routing
```

Comment:
I think we need to leave this section in the doc and just show the change in code to use it. We should also mention that other features can be accessed via the ProducerBuilderCustomizer such as interceptors.